File: save_game.cpp
class Mentes
{
  int game_save = 0;

public:
  void complete_level(int index)
  {
    game_save = game_save | (1 << index);
  }
  bool is_completed(int index)
  {
    return (game_save & (1 << index)) != 0;
  }
  void reset()
  {
    game_save = 0;
  }
};

File: save_game.java
public class Mentes {
    private int gameSave = 0;

    public void completeLevel(int index) {
        this.gameSave = gameSave | (1 << index);
    }

    public boolean isCompleted(int index) {
        return (gameSave & (1 << index)) != 0;
    }

    public void reset() {
        this.gameSave = 0;
    }
}